### PR TITLE
added blurb about docker open source program,

### PR DESCRIPTION
**Pull request by kapsakcj**
pull rate limits, hosting on quay.io to readme

Would love to hear your feedback on this before I merge in to the master branch

### CODE REVIEW

**Review by k-florek — APPROVED** [review]
Looks good to me! Thanks for this update. I would wait to merge this when we confirm the pull limits are raised and the autobuild feature is restored. I've been checking so I'll let you know when I see it.

**Comment by kapsakcj** [comment]
Thank you! 

Agreed, let's hold off until we hear back about those two points. Still haven't encountered docker pull limits in my Terra analyses with these containers and haven't heard of others experiencing them.

File: README.md
Comment (erinyoung) [inline README.md:20]: I think these lines are a little confusing. Are there pull limits?

I think something like 
> In November 2020, Docker began to implement pull rate limits for images hosted on dockerhub. This limits the number of `docker pull`'s per time period (e.g. anonymous user allows 100 pulls per six hours). We applied and were approved for Docker's "Open Source Program", which should have removed the pull rate limits for all the `staphb` docker images. If you encounter a "< insert pull rate error message text >", please let us know by submitting an [issue](https://github.com/StaPH-B/docker-builds/issues).

**Comment by erinyoung** [comment]
Sorry that it took me so long to read over this. I did NOT forget. I just wanted to spend some time to make sure it was accurate.

I checked out the two links, and they both appear to work.

I have been trying to trigger a pull rate limit error (which I often got with bcftools actually), but have been unable to.

**Comment by kapsakcj** [comment]
Thanks for the feedback, Erin. No worries on the wait. I updated the paragraph you commented on.

We are still waiting to hear back from Docker about the pull limits being removed as well as restoring the automated build function of dockerhub. I've had to manually `docker build` and `docker push` images to dockerhub recently.

I actually hit a couple pull limits last week and Kelsey mentioned she queried the dockerhub API and saw the limits were still in place.

Still planning to wait on merging this PR until we hear back

**Pull request merged**
